apps/ui/src/pages/index.tsx

```diff
@@ -16,27 +16,106 @@
  * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-import { Layout } from 'antd';
 
-import FooterComponent from '@/components/Footer';
-import HeaderComponent from '@/components/Header';
+import { AuditOutlined, FileOutlined, SignatureOutlined } from '@ant-design/icons';
+import { Avatar, Button, Col, Flex, Layout, Typography, theme } from 'antd';
 
 const { Content } = Layout;
+const { Title, Paragraph, Link, Text } = Typography;
+const { useToken } = theme;
 
-const contentStyle: React.CSSProperties = {
-	display: 'flex',
-	justifyContent: 'center',
-	alignItems: 'center',
-	textAlign: 'center',
-	backgroundColor: '#FFF0F6',
+const heroStyle: React.CSSProperties = {
+	width: '80%',
+	marginInline: 'auto',
+	minHeight: 400,
+	paddingBottom: 50,
+	paddingTop: 50,
 };
 
 export const HomePage = () => {
+	const { token } = useToken();
+
 	return (
-		<Layout>
-			<HeaderComponent />
-			<Content style={contentStyle}>PCGL Hero Content</Content>
-			<FooterComponent />
-		</Layout>
+		<Content>
+			<Flex className="hero-background-image" justify="center">
+				<Flex align="center" style={heroStyle}>
+					<Col span={12}>
+						<Flex vertical>
+							<Title style={{ color: token.colorTextSecondary }}> Apply for Access to Controlled Data</Title>
+							<Paragraph style={{ color: token.colorTextSecondary }}>
+								The PCGL Data Access Compliance Office (PCGL DACO) handles requests from scientists, researchers and
+								commercial teams for access to PCGL Controlled Data.
+							</Paragraph>
+							<Col span={6}>
+								<Button type="link" color="primary" variant="solid">
+									Get Started
+								</Button>
+							</Col>
+						</Flex>
+					</Col>
+				</Flex>
+			</Flex>
+			<Flex align="center" style={{ ...heroStyle }}>
```

Comment:
For consistency with the above `Flex` element this could just be `style={heroStyle}`